core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -1048,7 +1048,7 @@ export class WorkflowActionService {
 
       const operatorsAndPositions: { op: OperatorPredicate; pos: Point }[] = [];
       workflowContent.operators.forEach(op => {
-        const opPosition = workflowContent.operatorPositions[op.operatorID];
```

Comment:
why is this change needed?

Reply:
changed from const to let because I need to move the workflow